crates/pathfinder/src/rpc/v01/api.rs

```diff
@@ -384,10 +384,11 @@ impl RpcApi {
                 .map_err(internal_server_error)?;
 
             // There is a dedicated error code for a non-existent contract in the RPC API spec, so use it.
-            if contract_state_hash.0 == StarkHash::ZERO {
+            if contract_state_hash.is_none() {
                 return Err(Error::from(ErrorCode::ContractNotFound));
             }
 
+            let contract_state_hash = contract_state_hash.unwrap();
```

Comment:
These could be merged using `.ok_or` variant.

Reply:
Done